Examples/Tutorial/README.md

```diff
@@ -0,0 +1,27 @@
+Katydid Tutorial
```

Comment:
I think we should move the tutorial documentation to the main documentation area.  The README in this directory should direct people to there.